apm-sniffer/config/agent.config

```diff
@@ -78,3 +78,6 @@ logging.level=${SW_LOGGING_LEVEL:INFO}
 
 # mysql plugin configuration
 # plugin.mysql.trace_sql_parameters=${SW_MYSQL_TRACE_SQL_PARAMETERS:false}
+
+# influxdb plugin configuration
+# plugin.influxdb.trace_influxql=${SW_INFLUXDB_TRACE_INFLUXQL:true}
```

Comment:
This is not a widely used config, don't need to add it into the default agent.config. Putting it in the document should be enough.